src/content/developers/docs/gas/index.md

```diff
@@ -21,88 +20,113 @@ Since each Ethereum transaction requires computational resources to execute, eac
-| --- | ------------------: | ---------------------------: | ------------------: | ---------------: | --------------------------------------------------------------- |
-| A   |             90 gwei |                      90 gwei |                 N/A |              N/A | This transaction is not going in the block                      |
-| B   |            200 gwei |                       5 gwei |              5 gwei |         105 gwei | The priority fee is the maximum priority fee                    |
-| C   |            120 gwei |                      30 gwei |             20 gwei |         120 gwei | The priority fee is the maximum (total) fee minus the base fee  |
-| D   |            200 gwei |                     200 gwei |            100 gwei |         200 gwei | Transactions that specify gas price are charged the full amount |
+With the new base fee getting burned, the London Upgrade introduced a priority fee (tip) to incentivize miners to include a transaction in the block. Without tips, miners would find it economically viable to mine empty blocks, as they would receive the same block reward. Under normal conditions, a small tip provides miners a minimal incentive to include a transaction. For transactions that need to get preferentially executed ahead of other transactions in the same block, a higher tip will be necessary to attempt to outbid competing transactions.
 
-Miners and validators are expected to choose the transactions that will pay them the highest priority fees.
+### Max Fee {#maxfee}
 
-This mechanism is more complicated than the simple gas price auction, but it has the advantage of making gas fees more predictable, as well as making ETH more
-valuable by removing some of it from circulation. The maximum fee per gas functions as a [second price auction](https://oko.uk/blog/first-price-vs-second-price-auctions),
-which is more efficient than the previous mechanism that is a first price auction. Users can submit transactions with a much higher maximum fee per gas, corresponding
+To execute a transaction on the network users are able to specify a maximum limit they are willing to pay for their transaction to be executed. This optional parameter is known as the `maxFeePerGas`. In order for a transaction to be executed the max fee must exceed the sum of the base fee and the tip. The transaction sender is refunded the difference between the max fee and the sum of the base fee and tip.
+
+## EIP-1559 {#eip-1559}
+
+The implementation of [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) in the London Upgrade makes the transaction fee mechanism more complicated than the simple gas price auction, but it has the advantage of making gas fees more predictable, as well as making ETH more valuable by removing some of it from circulation. The base fee functions as a [second price auction](https://oko.uk/blog/first-price-vs-second-price-auctions),
```

Comment:
A few nits here :-) 
```suggestion
The implementation of [EIP-1559](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) in the London Upgrade makes the transaction fee mechanism more complex than the current gas price auction, but it has the advantage of making gas fees more predictable, resulting in a more efficient transaction fee market. 
```

I don't think it is a good idea to imply that 1559 will "make ETH more valuable", given how many other factors affect the price of cryptocurrency. 

Additionally, the base fee doesn't make the network use a second price auction. 2nd price auctions require a trusted third party, which we can't have in Ethereum. The `baseFee` is our way of gauging demand for block space (and subsequently pricing that block space) _without_ a 2nd price auction.